lib/Site.js

```diff
@@ -395,6 +401,7 @@ Site.prototype.generate = function (baseUrl) {
       .then(() => this.buildSourceFiles())
       .then(() => this.copyMarkBindAsset())
       .then(() => this.writeSiteData())
+      .then(() => logger.info(`Build end. building time: ${(new Date() - startTime) / 1000}s`))
```

Comment:
Would be better to move the calculation out of the string interpolation to make things clearer.

I improved the message also:

```javascript
.then(() => {
    const endTime = new Date();
    const totalBuildTime = (endTime - startTime) / 1000;
    logger.info(`Website generation complete! Total build time: ${totalBuildTime}s`));
}
```